src/components/Button.tsx

```diff
@@ -169,7 +169,11 @@ export default function ({
     >
       {loading && <Loading small={small} />}
       {typeof children === 'string' ? <span>{children}</span> : children}
-      {withArrow && <Arrow disabled={disabled || loading} />}
+      {withArrow && (
+        <div className={width('w-4')}>
```

Comment:
The arrow has different sizes, so I removed `width` and `height` from `svg` and wrapped it, where we use it